docs/xalgo.md

```diff
@@ -119,13 +119,20 @@ made knowing only the id. The scheme is structured as follows:
   can be either S, R or T (signifying System, Rule or Table)
 
 - [1]: One of (A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q). This refers to the
-  [ISIC
-  Rev.3](https://unstats.un.org/unsd/cr/registry/regcst.asp?Cl=2)
+  [ISIC Rev.3](https://unstats.un.org/unsd/cr/registry/regcst.asp?Cl=2)
   category for the component.
   
-- [2-6]: Any valid ASCII character except space available on an
+- [2-21]: These next locations in a Rule Id are reserved for references to globally-
```

Comment:
Adding more space is fine, but I don't like this idea of control characters in the naming. It's awkward to do this and ... I don't really see the point.